### ISSUE TITLE
Implement JpqlSerializer for JpqlMax

### ISSUE THREAD

**Issue by shouwn** [line/kotlin-jdsl#342]

**Comment by jbl428** [comment]
@shouwn 
Could i take this issue? 
I have found that `JpqlMaxSerializer` already exist.
and serializers for `SUM`, `AVG`, `MIN` can be implemented like this file.
so i think i can handle all issues. :)

```
aggregate_expression ::= { AVG | MAX | MIN | SUM } ([DISTINCT] state_field_path_expression) | COUNT ([DISTINCT] identification_variable | state_field_path_expression | single_valued_association_path_expression)
```

**Comment by shouwn** [comment]
Hi @jbl428 You can take this issue!

When I defined the JpqlSerializer interface, the code that I was testing in local on how the actual implementation could be written seems to have been mixed up.

I will modify the assignee. I welcome your contribution!

**Comment by shouwn** [comment]
Could you comment on AVG, MIN, and SUM issues? I tried to modify the assignee of those issues as you, but it didn't show up in the selections.

**Comment by shouwn** [comment]
This issue is closed by #396
